File: ToyInterpreter 2/src/mobserver/VarTableObserver.java
package mobserver;

import Model.IDictionary;

public class VarTableObserver extends CObserver implements java.io.Serializable{
	@Override
	public void onUpdate(Object o) {
		System.out.println("Var table:\n");
		System.out.println(o.toString());
	}
}

File: ToyInterpreter/ToyInterpreter/Model/MObserver/VarTableObserver.cs
using System;

namespace ToyInterpreter.Model.MObserver
{
	[Serializable()]
	public class VarTableObserver : CObserver
	{
		public override void onUpdate (Object o)
		{
			Console.WriteLine ("VarTable:");
			Console.WriteLine (o.ToString());
		}
	}
}

